restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/Starting Restcomm-Connect.adoc

```diff
@@ -69,7 +90,7 @@ Then you should change the default password.
 
 * If you need to stop the Restcomm you need to run *$RESTCOMM_HOME/bin/restcomm/stop-restcomm.sh* command.
 
-*Step 4 – Making Test SIP Calls using the Demo Apps*
+== Making Test SIP Calls using the Demo Apps
```

Comment:
Let's avoid referencing a third party SIP Phone as developers have a lot of issues in setting up SIP Phones. Let's guide them to use Olympus and redirect to http://docs.telestax.com/how-to-use-olympus-with-restcomm/ or a new article.